src/madpack/madpack.py

```diff
@@ -357,7 +356,7 @@ def _get_rev_num(rev):
         @param rev version text
     """
     try:
-        num = re.findall('[0-9]', rev)
```

Comment:
The primary bug in `re.findall('[0-9]', rev)` was that a version string like `1.3.10` resulted in output `[1, 3, 1, 0]` instead of `[1, 3, 10]`. This PR solely aims to fix that. Ideally I would like to use something like `distutils.version.LooseVersion` to compare versions, but we also need the version parsed in a list and I'm not aware if `LooseVersion` provides that.